tests/core/test_lane.py

```diff
@@ -4183,7 +4178,8 @@ def test_groups(
 
         # "Children", which will contain one book, the one with audience children and defined target age.
         children = db.lane("Children")
-        children.audience = Classifier.AUDIENCE_CHILDREN
+        children.audiences = Classifier.AUDIENCE_CHILDREN
+        children.target_age = (0, 4)
```

Comment:
The filter object seems to now __need__ a target age range to apply age ranges now.